### PR TITLE
kdown

### PR DESCRIPTION
**Pull request by utkarshsharma00**
This program finds all those nodes which are 'k' down from the root node of the tree.

**Fixes issue:**

**Changes:**

### CODE REVIEW

File: code/data_structures/src/tree/binary_tree/binary_tree/kdown [hunk 1]
```diff
@@ -0,0 +1,84 @@
+#include <iostream>
```
Comment (arnavb) [inline code/data_structures/src/tree/binary_tree/binary_tree/kdown]: Missing file extension.

File: code/data_structures/src/tree/binary_tree/binary_tree/kdown [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+#include <vector>
+
+using namespace std;
```
Comment (arnavb) [inline code/data_structures/src/tree/binary_tree/binary_tree/kdown]: Don't use this.

File: code/data_structures/src/tree/binary_tree/binary_tree/kdown [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+{
+public:
+    int data;
+    Node *left;
+    Node *right;
+
+    Node(int data)
+    {
+        this->data = data;
+        this->left = NULL;
```
Comment (arnavb) [inline code/data_structures/src/tree/binary_tree/binary_tree/kdown]: `nullptr`.

File: code/data_structures/src/tree/binary_tree/binary_tree/kdown [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+        this->right = NULL;
+    }
+};
+
+Node *construct(vector<int> &datalist)
```
Comment (arnavb) [inline code/data_structures/src/tree/binary_tree/binary_tree/kdown]: Use a class that represents a binary tree, not standalone functions.

File: code/data_structures/src/tree/binary_tree/binary_tree/kdown (hunk 1 above)
Comment (utkarshsharma00) [reply code/data_structures/src/tree/binary_tree/binary_tree/kdown]: Hey, this pull request has the wrong code. Kindly elaborate on your previous comments in pull request #4051

Comment (arnavb) [reply code/data_structures/src/tree/binary_tree/binary_tree/kdown]: @utkarshsharma00 Same problems in your other PR. If this one isn't valid, close it. But implement the changes here in that PR.

Comment (utkarshsharma00) [reply code/data_structures/src/tree/binary_tree/binary_tree/kdown]: @arnavb I am not able to understand the changes that you want me to do. I am not able to get these comments of yours "Missing file extension" and "Don't use this".

**Pull request closed without merging**